dist/TempManager.js

```diff
@@ -0,0 +1,31 @@
+class TempManager {
+  constructor() {
+    this.cityData = [];
+  }
+
+  async getDataFromDB() {
+    let citiesData = await $.get(`/cities`);
+    this.cityData = citiesData;
+    // citiesData.forEach(d => this.cityData.push(d))
+  }
+
+  async getCityData(cityName) {
+    let cityData = await $.get(`/city/${cityName}`);
+    this.cityData.push(cityData);
+  }
+
+  async saveCity(cityName, index) {
+    let postBodyData = this.cityData.find(c => c.name == cityName);
+    await $.post(`/city`, postBodyData);
+    this.cityData[index]._id = true
```

Comment:
The _id value isn't a boolean. you should receive it from the server after saving the city into the DB.